object/tombstone.go

```diff
@@ -130,7 +157,11 @@ func (t *Tombstone) Marshal() []byte {
 //
 // See also [Tombstone.Marshal].
 func (t *Tombstone) Unmarshal(data []byte) error {
-	return (*tombstone.Tombstone)(t).Unmarshal(data)
+	var m tombstone.Tombstone
+	if err := m.Unmarshal(data); err != nil {
```

Comment:
Shouldn't `Unmarshal` guarantee us `m` consistency on its own?

Reply:
it does not. For example, IDs are raw bytes in protocol, but particular arrays in current layer

Reply:
OK.